data/fixtures/scopes/typescript.core/value.typeAlias.scope

```diff
@@ -0,0 +1,20 @@
+type Aaa = Bbb;
+---
+
+[Content] = 0:11-0:14
+             >---<
+0| type Aaa = Bbb;
+
+[Removal] = 0:10-0:14
+            >----<
+0| type Aaa = Bbb;
```

Comment:
I'm pretty sure `type Aaa;` would not be valid, so I don't think there's any point in supporting "chuck value"